item.rb

```diff
@@ -0,0 +1,28 @@
+require 'securerandom'
+require 'date'
+
+class Item
+  attr_accessor :label, :genre, :author
+  attr_reader :id, :published_date, :archived
+
+  def initialize(published_date = nil, id: nil)
+    @id = id || SecureRandom.uuid
+    @label = nil
+    @genre = nil
+    @author = nil
+    @published_date = published_date
+    @archived = false
+    move_to_archive
+  end
+
+  def can_be_archived?
+    return false if @published_date.nil?
+
+    published_date = DateTime.parse(@published_date)
+    published_date < DateTime.now - 30
+  end
```

Comment:
- [ ] As required for the project the Item class to have a `can_be_archived?()` method  returning true if `published_date` is older than 10 years, otherwise, to return false, I think it should better if you update the above to match the expected result.
```ruby
# example
def can_be_archived?
    current_year = Time.now.year
    publication_year = @published_date.year

    if current_year - publication_year > 10
      return true
    else
      return false
    end
  end
```